### PR TITLE
Added connection timeout. Fixes #7417

### LINKED ISSUE #7417: [Bug] Invoke-DbaDbDataMasking - Parameter ConnectionTimeout (and others?) not used
Hi @sanderstad , hava a look at Invoke-DbaDbDataMasking: I don't see $ConnectionTimeout anywhere beeing used. Have not checked the other parameters...

### PR DESCRIPTION
**Pull request by sanderstad**
Added splatting fof command for readability

## Type of Change

 - [X] Bug fix (non-breaking change, fixes #7417 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

### Purpose
The invoke command the connection timeout to make sure that commands would not execute for a long period of time

### Approach
Added variable to several query commands

### Commands to test
Invoke-DbaDbDataMasking

### CODE REVIEW

File: functions/Invoke-DbaDbDataMasking.ps1 [hunk 1]
```diff
@@ -367,7 +367,7 @@ function Invoke-DbaDbDataMasking {
 
                         $query = "CREATE NONCLUSTERED INDEX [$($maskingIndexName)] ON [$($dbTable.Schema)].[$($dbTable.Name)]([$($identityColumn)])"
 
-                        Invoke-DbaQuery -SqlInstance $server -SqlCredential $SqlCredential -Database $db.Name -Query $query
+                        Invoke-DbaQuery -SqlInstance $server -SqlCredential $SqlCredential -Database $db.Name -Query $query -QueryTimeout
```
Comment (andreasjordan) [inline functions/Invoke-DbaDbDataMasking.ps1:370]: Is there a ` $ConnectionTimeout` missing at the end?

Comment (andreasjordan) [reply functions/Invoke-DbaDbDataMasking.ps1:370]: Or better `$CommandTimeout`?

**Comment by andreasjordan** [comment]
Why do you use `$ConnectionTimeout` for `-QueryTimeout` of `Invoke-DbaQuery`? You also have `$CommandTimeout`which you don not use at all and wich would be a better fit. And have a look at the documenation of the two parameters: They are the same.

File: functions/Invoke-DbaDbDataMasking.ps1 (hunk 1 above)
Comment (andreasjordan) [reply functions/Invoke-DbaDbDataMasking.ps1:370]: This is probably the reason the tests fail.

**Comment by sanderstad** [comment]
@andreasjordan You are correct, there was an obsolete parameter.
I removed the connectiontimeout parameter and replaced everything with the command timeout.
I'm not sure why I put that in, maybe to distinquish the connection to the servers and the query timeouts separately

**Review by andreasjordan — APPROVED** [review]

**Comment by potatoqualitee** [comment]
🙌🏼  awesome, thank you both!

**Pull request merged**